File: WeatherApp 1/WeatherApp/Coordinates.m
//
//  Location.m
//  WeatherApp
//
//  Created by projas on 2/11/16.
//  Copyright © 2016 projas. All rights reserved.
//

#import "Coordinates.h"

@interface Coordinates ()

@end

@implementation Coordinates

- (instancetype)initWithLatitude:(NSString*)latitude
                       longitude:(NSString*)longitude
{
    self = [super init];
    if (self) {
        self.latitude = latitude;
        self.longitude = longitude;
    }
    return self;
}

@end

File: WeatherApp-Swift with Request/WeatherApp-Swift/Coordinates.swift
//
//  Coordinates.swift
//  WeatherApp-Swift
//
//  Created by projas on 3/28/16.
//  Copyright © 2016 projas. All rights reserved.
//

import UIKit
import CoreLocation

class Coordinates: NSObject {
    var latitude:   String
    var longitude:  String
    
    init(WithLatitude latitude: String, longitude: String) {
        self.latitude = latitude
        self.longitude = longitude
    }
    
}
